dependency_support/com_google_skywater_pdk/declare_cell_library.bzl

```diff
@@ -51,7 +51,17 @@ def declare_cell_library(workspace_name, name):
         )
     skywater_cell_library(
         name = name,
+        srcs = native.glob(
+            include = [
+                "cells/**/*.lef",
+            ],
+            exclude = [
+                "cells/**/*.magic.lef",
```

Comment:
Adding a comment explaining why this is excluded would help future reader's understanding of this code I think.